tests/test_module_state.py

```diff
@@ -7,6 +7,11 @@
 
 from .conftest import TEST_PLAYBOOKS
 
+try:
+    ast_try = ast.TryExcept
+except AttributeError:
+    ast_try = ast.Try
```

Comment:
I think you can use six for things like this.

Reply:
you mean `if six.PY2` things? yeah. I'm too used to not use six for "reasons" :)